src/main/java/org/broadinstitute/hellbender/utils/hdf5/HDF5PoN.java

```diff
@@ -274,14 +278,16 @@ public void setTargetFactors(final RealMatrix targetFactors) {
 
     @Override
     public RealMatrix getNormalizedCounts() {
-        return readMatrixAndCheckDimensions(NORMALIZED_PCOV_PATH, targetNames.get().size(),
-                sampleNames.get().size());
+        // Note the check is using sample names as number of rows and targets as number of columns....
```

Comment:
Be explicit and say how this comment is a consequence of the transposed storage.